content/geometry/general.adoc

```diff
@@ -134,18 +134,29 @@ This is an example of a 3D model file in glTF format with an accompanying 3D ass
 
 == Requirements
 
-Materials shall be separated.
-
-NOTE: TODO: Extend rule and find better location. What do you want to express, that two
-materials with different properties need to be modeled by separate objects?
-Are there any other requirements regarding quality, semantic/animation/material structure?
+* The object's geometry shall be in real word scale, using meters as unit for measurement for distances.
+* The object's origins and pivot points shall coincide with position and orientation of the origin of the object's reference coordinate frame if not specified otherwise.
+* Meshes shall not contain problematic characteristics such as doubled/isolated/coincident/coplanar/degenerate primitives.
+* Meshes shall have outside facing normals. Soft or hard edges shall be set correctly.
+* Meshes shall not be empty, nor contain multiple LODs.
+* Meshes shall be triangulated and potential normal maps shall match that triangulation.
+* Rendering materials shall support PBR workflows and there shall not be geometry without assigned material.
+* Additional requirements apply, when OpenMATERIAL 3D assignment textures are used:
+** UVs shall not overlap, they shall be within 0-1 UV space and channel 1 shall be used for assignment textures.
```

Comment:
"UVs shall not overlap, they shall be within 0-1 UV" - I would see this as an recommendation and not an requirement, because overlapping UVs and extending UV space can be useful, depending on the use case.

Reply:
Would be fine for me to move to the Recommendations. @LudwigFriedmann @MatthiasThDs who do you see it?

Reply:
yes I agree.
I thought about things that make sense most of the times. But of course, sensible exceptions should not breach the requirements.